validphys2/src/validphys/photon/compute.py

```diff
@@ -0,0 +1,205 @@
65e-03, 5.14175977083962e-03, 6.41865096062317e-03, 7.95137940306351e-03, 9.76689999624100e-03, 1.18876139251364e-02, 1.43298947643919e-02, 1.71032279460271e-02, 2.02100733925079e-02, 2.36463971369542e-02, 2.74026915728357e-02, 3.14652506132444e-02, 3.58174829282429e-02, 4.04411060163317e-02, 4.53171343973807e-02, 5.04266347950069e-02, 5.57512610084339e-02, 6.12736019390519e-02, 6.69773829498255e-02, 7.28475589986517e-02, 7.88703322292727e-02, 8.50331197801452e-02, 9.13244910278679e-02, 9.77340879783772e-02, 1.04252538208639e-01, 1.10871366547237e-01, 1.17582909372878e-01, 1.24380233801599e-01, 1.31257062945031e-01, 1.38207707707289e-01, 1.45227005135651e-01, 1.52310263065985e-01, 1.59453210652156e-01, 1.66651954293987e-01, 1.73902938455578e-01, 1.81202910873333e-01, 1.88548891679097e-01, 1.95938145999193e-01, 2.03368159629765e-01, 2.10836617429103e-01, 2.18341384106561e-01, 2.25880487124065e-01, 2.33452101459503e-01, 2.41054536011681e-01, 2.48686221452762e-01, 2.56345699358723e-01, 2.64031612468684e-01, 2.71742695942783e-01, 2.79477769504149e-01, 2.87235730364833e-01, 2.95015546847664e-01, 3.02816252626866e-01, 3.10636941519503e-01, 3.18476762768082e-01, 3.26334916761672e-01, 3.34210651149156e-01, 3.42103257303627e-01, 3.50012067101685e-01, 3.57936449985571e-01, 3.65875810279643e-01, 3.73829584735962e-01, 3.81797240286494e-01, 3.89778271981947e-01, 3.97772201099286e-01, 4.05778573402340e-01, 4.13796957540671e-01, 4.21826943574548e-01, 4.29868141614175e-01, 4.37920180563205e-01, 4.45982706956990e-01, 4.54055383887562e-01, 4.62137890007651e-01, 4.70229918607142e-01, 4.78331176755675e-01, 4.86441384506059e-01, 4.94560274153348e-01, 5.02687589545177e-01, 5.10823085439086e-01, 5.18966526903235e-01, 5.27117688756998e-01, 5.35276355048428e-01, 5.43442318565661e-01, 5.51615380379768e-01, 5.59795349416641e-01, 5.67982042055800e-01, 5.76175281754088e-01, 5.84374898692498e-01, 5.92580729444440e-01, 6.00792616663950e-01, 6.09010408792398e-01, 6.17233959782450e-01, 6.25463128838069e-01, 6.33697780169485e-01, 6.41937782762089e-01, 6.50183010158361e-01, 6.58433340251944e-01, 6.66688655093089e-01, 6.74948840704708e-01, 6.83213786908386e-01, 6.91483387159697e-01, 6.99757538392251e-01, 7.08036140869916e-01, 7.16319098046733e-01, 7.24606316434025e-01, 7.32897705474271e-01, 7.41193177421404e-01, 7.49492647227008e-01, 7.57796032432224e-01, 7.66103253064927e-01, 7.74414231541921e-01, 7.82728892575836e-01, 7.91047163086478e-01, 7.99368972116378e-01, 8.07694250750291e-01, 8.16022932038457e-01, 8.24354950923382e-01, 8.32690244169987e-01, 8.41028750298844e-01, 8.49370409522600e-01, 8.57715163684985e-01, 8.66062956202683e-01, 8.74413732009721e-01, 8.82767437504206e-01, 8.91124020497459e-01, 8.99483430165226e-01, 9.07845617001021e-01, 9.16210532771399e-01, 9.24578130473112e-01, 9.32948364292029e-01, 9.41321189563734e-01, 9.49696562735755e-01, 9.58074441331298e-01, 9.66454783914439e-01, 9.74837550056705e-01, 9.83222700304978e-01, 9.91610196150662e-01, 1.00000000000000e+00])
```

Comment:
This grid exists in writer.py so rather use that than create another copy of it, though note that #1537 also moves it

Reply:
But instead of using the same grid for the interpolator wouldn't it be better to autogenerate one using logspace up to 0.1 and linspace up to 1 or something instead of hardcode it?

Reply:
Yeah, the same discussion went on with @andreab1997, but I thought that it was inaccessible to vp, since the evolven3fit was mostly standalone.
If it is already available to vp, of course reuse the already included one.

Reply:
I think this grid is pretty close to that. I would be fine either way, the output should not be significantly be affected.

Reply:
Also, during development I'd recommend using a small one so you can iterate results more quickly.

Reply:
Yeah, it is `eko.lambertgrid`, but we 200 points it doesn't matter so much (200 points are definitely too many, but it looks like we do not have the freedom to change).
However, instead of a dump of 200 points we should use either `np.logspace` or `eko.lambertgrid`.

Reply:
> I think this grid is pretty close to that.

It is not pretty close, it is the exact same. They are both coming from `eko.lambertgrid`.

Reply:
How is `eko.labertgrid` defined? Exactly linear + log described by @scarlehoff ?

Reply:
@scarlehoff I wanted to use the output grid so that the final PDF that will be the output of the fit will not be an interpolation, but the exact values. Obviously we can use the grid you are proposing but this will slightly increase the error.

Reply:
@AleCandido I checked an the grid that I'm defining here is not exactly the same of `make_labert_grid` from `eko`.

Reply:
If that's the motivation that's ok. But then it is better if you import it from `writer.py` (or from wherever it is now, I don't know whether it changed) so that if the output grid changes, it also changes here.

Still, I would use for development something like 50 points just so it is faster :P

Reply:
> If that's the motivation that's ok. But then it is better if you import it from writer.py (or from wherever it is now, I don't know whether it changed) so that if the output grid changes, it also changes here.

@andreab1997 moved it but it's not yet merged. Also, do we care that it is defined in n3fit and this is vp?

Reply:
> If that's the motivation that's ok. But then it is better if you import it from `writer.py` (or from wherever it is now, I don't know whether it changed) so that if the output grid changes, it also changes here.
> 
> Still, I would use for development something like 50 points just so it is faster :P

Yes I agree both points